<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>search</class>
 <widget class="QWidget" name="search">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>689</width>
    <height>493</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Search</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>1</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="options">
     <property name="styleSheet">
      <string notr="true">QWidget{
background-color:#64697D;
color:#000000; 
border: 1px rgb(67, 67, 67);
}</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QFrame" name="searchbar">
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>5</number>
         </property>
         <property name="rightMargin">
          <number>5</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QPushButton" name="title">
           <property name="font">
            <font>
             <pointsize>16</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">Text-align:left
</string>
           </property>
           <property name="text">
            <string notr="true">Search</string>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/Search.svg</normaloff>:/icons/Search.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="flat">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLineEdit" name="searchFF">
           <property name="styleSheet">
            <string notr="true">QLineEdit {
    border: 1px solid #ffffff;
    padding: 5px 5px 5px 10px;
}
</string>
           </property>
           <property name="clearButtonEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="findCMD">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
Text-align:left;
color:#E6E6E6; 
padding: 5px 5px 5px 5px;
}

QPushButton:pressed{
background-color:#1793D1; 
}

QPushButton:hover {
background-color: #1793D1;
}</string>
           </property>
           <property name="text">
            <string>Find</string>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="locateCMD">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
Text-align:left;
color:#E6E6E6; 
padding: 5px 5px 5px 5px;
}

QPushButton:pressed{
background-color:#1793D1; 
}

QPushButton:hover {
background-color: #1793D1;
}</string>
           </property>
           <property name="text">
            <string>Locate</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="cancelProc">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>12</pointsize>
            </font>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton{
Text-align:left;
color:#E6E6E6; 
padding: 5px 5px 5px 5px;
}

QPushButton:pressed{
background-color:#1793D1; 
}

QPushButton:hover {
background-color: #1793D1;
}</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="more">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">
QPushButton:checked{
background-color:#45454D ;
}

QPushButton:!checked{
color: #57585D ;
}
</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/down_w.svg</normaloff>:/icons/down_w.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
        <zorder>more</zorder>
        <zorder>title</zorder>
        <zorder>findCMD</zorder>
        <zorder>locateCMD</zorder>
        <zorder>searchFF</zorder>
        <zorder>cancelProc</zorder>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="pathfream">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>48</height>
         </size>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="folderPath">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLineEdit{
border: 1px solid #ffffff;
padding: 5px 5px 5px 10px;
}

QLineEdit:focus{
border:1px solid gray;
}</string>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="placeholderText">
            <string>Add folder path (default is root)...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="setfolder">
           <property name="focusPolicy">
            <enum>Qt::NoFocus</enum>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="icon">
            <iconset resource="../icons.qrc">
             <normaloff>:/icons/folder_w.svg</normaloff>:/icons/folder_w.svg</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>30</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="typeframe">
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QToolButton" name="typeAll">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
color : #E6E6E6;
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QToolButton:checked
{
border:none;
border-bottom: 2px solid #DCDCDC;
color: #DCDCDC; 
font-weight: bold;
}
</string>
           </property>
           <property name="text">
            <string>All</string>
           </property>
           <property name="iconSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
           <property name="autoRaise">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="typePicture">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
color : #E6E6E6;
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QToolButton:checked
{
border:none;
border-bottom: 2px solid #DCDCDC;
color: #DCDCDC; 
font-weight: bold;
}
</string>
           </property>
           <property name="text">
            <string>Picture</string>
           </property>
           <property name="iconSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="typeFolder">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
color : #E6E6E6;
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QToolButton:checked
{
border:none;
border-bottom: 2px solid #DCDCDC;
color: #DCDCDC; 
font-weight: bold;
}
</string>
           </property>
           <property name="text">
            <string>Folder</string>
           </property>
           <property name="iconSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="typeMedia">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
color : #E6E6E6;
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QToolButton:checked
{
border:none;
border-bottom: 2px solid #DCDCDC;
color: #DCDCDC; 
font-weight: bold;
}
</string>
           </property>
           <property name="text">
            <string>Media</string>
           </property>
           <property name="iconSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QToolButton" name="typeother">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>50</weight>
             <bold>false</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QToolButton{
color : #E6E6E6;
border: 1px rgb(67, 67, 67);
padding: 5px 5px 5px 5px;
}

QToolButton:checked
{
border:none;
border-bottom: 2px solid #DCDCDC;
color: #DCDCDC; 
font-weight: bold;
}
</string>
           </property>
           <property name="text">
            <string>Other</string>
           </property>
           <property name="iconSize">
            <size>
             <width>70</width>
             <height>80</height>
            </size>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextOnly</enum>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="itemCount">
           <property name="font">
            <font>
             <pointsize>11</pointsize>
            </font>
           </property>
           <property name="text">
            <string>0 item(s) found</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="infoPage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="fonudpage">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableWidget" name="results">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QTableView::item{
border-bottom-style:solid;
border-width:1;
border-color: #E6E6E6;
padding: 0px 20px 0px 20px;
}

QTableView::item:selected {
background-color:#1793D1 ;
color:#000000; 
}

QTableView {
background-color:#F2F2F2 ;
padding: 0px 0px 20px 0px;
color:#000000; 
}

QHeaderView{ 
height: 50px;
background-color:#E6E6E6 ;
color:#000000;
}

QHeaderView::section { 
background-color:#E6E6E6;
color:#000000;
border: 1px #ffffff;
padding-top: 4px;
font: 12pt &quot;Cantarell&quot;;
}


QScrollBar:vertical {
background-color: rgba( 0, 0, 0, 72% );
max-width: 8px;
border-radius: 4px;
padding: 1px; }

QScrollBar::handle:vertical {
background-color: silver;
max-width: 6px;
border-radius: 3px; }

QScrollBar::sub-page:vertical {
background-color: rgba( 0, 0, 0, 72% ); }

QScrollBar::add-page:vertical {
background-color: rgba( 0, 0, 0, 72% ); }

QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical {
width: 0px; height: 0px; }

</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="gridStyle">
          <enum>Qt::NoPen</enum>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>180</number>
         </attribute>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string>Name</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Path</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
          <property name="textAlignment">
           <set>AlignCenter</set>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="notfoundpage">
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color:#F2F2F2 ;
color:#000000; 
border: 1px rgb(67, 67, 67);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="spacing">
        <number>20</number>
       </property>
       <property name="leftMargin">
        <number>100</number>
       </property>
       <property name="topMargin">
        <number>50</number>
       </property>
       <property name="rightMargin">
        <number>100</number>
       </property>
       <item>
        <widget class="QLabel" name="status">
         <property name="font">
          <font>
           <pointsize>36</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="activity">
      <property name="styleSheet">
       <string notr="true">QWidget{
background-color:#F2F2F2 ;
color:#000000; 
border: 1px rgb(67, 67, 67);
}</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="spacing">
        <number>15</number>
       </property>
       <property name="leftMargin">
        <number>45</number>
       </property>
       <property name="topMargin">
        <number>10</number>
       </property>
       <property name="rightMargin">
        <number>10</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Search Activity</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTreeWidget" name="activityList">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="styleSheet">
          <string notr="true">QScrollBar:vertical {
background-color: rgba( 0, 0, 0, 72% );
max-width: 8px;
border-radius: 4px;
padding: 1px; }

QScrollBar::handle:vertical {
background-color: silver;
max-width: 6px;
border-radius: 3px; }

QScrollBar::sub-page:vertical {
background-color: rgba( 0, 0, 0, 72% ); }

QScrollBar::add-page:vertical {
background-color: rgba( 0, 0, 0, 72% ); }

QScrollBar::add-line:vertical, 
QScrollBar::sub-line:vertical {
width: 0px; height: 0px; }

</string>
         </property>
         <property name="animated">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
         <attribute name="headerVisible">
          <bool>false</bool>
         </attribute>
         <column>
          <property name="text">
           <string notr="true">1</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>searchFF</tabstop>
  <tabstop>findCMD</tabstop>
  <tabstop>typeAll</tabstop>
  <tabstop>typePicture</tabstop>
  <tabstop>typeFolder</tabstop>
  <tabstop>typeMedia</tabstop>
  <tabstop>typeother</tabstop>
  <tabstop>results</tabstop>
  <tabstop>more</tabstop>
  <tabstop>folderPath</tabstop>
  <tabstop>setfolder</tabstop>
  <tabstop>locateCMD</tabstop>
  <tabstop>title</tabstop>
 </tabstops>
 <resources>
  <include location="../icons.qrc"/>
 </resources>
 <connections/>
</ui>
